sdks/go/pkg/beam/runners/direct/direct.go

```diff
@@ -65,6 +65,7 @@ func Execute(ctx context.Context, p *beam.Pipeline) (beam.PipelineResult, error)
 	if err != nil {
 		return nil, errors.Wrap(err, "translation failed")
 	}
+	beam.PipelineOptions.LoadOptionsFromFlags(make(map[string]bool))
```

Comment:
I just checked the go playground and at least at 1.17 it seems like lookups in nil maps do not panic (they just return the default value for the type). Prefer just passing in a nil instead, and if uncomfortable relying on the default behavior, initializing it in the function instead.

Insertions to nil maps *do* panic though.

https://go.dev/play/p/H7QiTyKeAyU

Reply:
Oh nice - that's much cleaner, thanks for calling that option out! Updated!